projects/plugins/jetpack/extensions/blocks/contact-form/editor.scss

```diff
@@ -172,6 +171,19 @@ input.components-text-control__input {
 	line-height: 16px;
 }
 
+/*
+	Gutenberg adds some default styling to the TextControl component
+	(https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_mixins.scss#L130)
+	For the TextareaControl, since the base styles are added to the StyledComponent,
+	(https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/textarea-control/styles/textarea-control-styles.ts#L13)
+	The output CSS has a lower specificity and it's being overridden on themes like
+	Twenty Twenty-One and Twenty Twenty-Two.
+	Adding this to normalize the styles between the TextControl and TextareaControl
+ */
+:where(.jetpack-field) textarea.components-textarea-control__input {
```

Comment:
Also, where are those values (`1px solid #757575`) are coming from? Asking merely to check if there are vars for them.

Reply:
I'm using `:where` because I don't want to increase the selector specificity.
If I nest `textarea.components-textarea-control__input` inside the `.jetpack-field {` it'll increase the specificity making it harder to themes override these styles.

> Also, where are those values (1px solid #757575) are coming from? Asking merely to check if there are vars for them.

It comes from [Gutenberg](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_mixins.scss#L130). There's no variable with this color available for use.